babybuddy/settings/docker.py

```diff
@@ -4,13 +4,14 @@
 # Database
 # https://docs.djangoproject.com/en/3.0/ref/settings/#databases
 
+# Load settings from env file / variables with fallback defaults to support current psql deployment
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
-        'NAME': 'postgres',
-        'USER': 'postgres',
+        'NAME': os.getenv('POSTGRES_DB') if os.getenv('POSTGRES_DB') else 'postgres',
```

Comment:
Lets use the `or` keyword here instead of inline `if-else` (also for L12,14-15).

E.g. `os.environ.get('POSTGRES_DB') or 'postgres'`

Reply:
Updated in 6f83e16. Also added in check for DB_ENGINE.

Reply:
Marking this unresolved as I just thought of another problem / question - are you going to put the exact string require by django for the db_engine type (i.e. django.db.backends.mysql) in the documentation for users to fill out or would there be an elegant way to check the variable and select the right engine string based on easier expected variables (i.e. mysql)?

Reply:
I think for a first pass, lets just have the let users fill in the `DB_ENGINE` string as desired. The Docker settings file should default to the currently value (`django.db.backends.postgresql_psycopg2`) and the documentation can be updated to reference https://docs.djangoproject.com/en/3.0/ref/settings/#engine for other standards.

Reply:
Updates pushed in 0c05cd3 with README.md changes reflected